pkg/daemon/daemon.go

```diff
@@ -13,8 +14,6 @@ import (
 	"strings"
 	"time"
 
-	"github.com/coreos/go-systemd/login1"
-	"github.com/coreos/go-systemd/sdjournal"
```

Comment:
can we `rm -rf vendor/github.com/coreos/go-systemd` now?

Reply:
or run `dep ensure` (but not sure it can cleanup unused vendors)

Reply:
No, it's still needed by Ignition.